### ISSUE TITLE
2021-9-16

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#606]
3816.continually add new prediction tasks by adapting shared parameters without access to training data for previously learned tasks; (2021-8-26)
3827.Using only examples for the new task; similar to joint training, except that our method does not need the old task’s images and labels; Learning without Forgetting approach has several advantages: 
3838.(1) Classification performance; (2) Computational efficiency; (3) Simplicity in deployment; (2021-8-27)
384II.RELATED WORK
3851.Multi-task learning, transfer learning; combination of Distillation Networks and fine-tuning; Distillation Networks, learn parameters in a simpler network that produce the same outputs as a more complex; (2021-8-27)
386II.1 Compared methods
3871.Feature Extraction; uses a pre-trained deep CNN to compute features for an image; these features are not specialized for the new task; (2021-8-27)
3882.Fine-tuning; low learning rate is to preserve; preserve representations that are important for the original task; (2021-8-28)
3893.Multitask learning; improve all tasks simultaneously by combining the common knowledge from all tasks; requires data from all tasks to be present, while our method requires only data for the new tasks; (2021-8-28)
